README.md

```diff
@@ -3,3 +3,37 @@
 We put people first and do our best to recognize, appreciate and respect the diversity of our global contributors. The MDN Web Docs project welcomes contributions from everyone who shares our goals and wants to contribute in a healthy and constructive manner within our community.
 
 MDN community is the repository where you will find the [offical MDN Web Docs public feedback discussions](https://github.com/mdn/mdn-community/discussions).
+
+## Some participation guideline highlights
+
+To ensure we maintain a healthy, friendly, and welcoming community, we want to highlight the behaviours we expect from our community:
+
+### Expected behaviours
+
+* **Be Respectful**
```

Comment:
```suggestion
* **Being respectful**
```
- to follow sentence casing, as is done for the content on MDN
- also changed 'be' from imperative to gerund form for it to flow with the title of 'Expected behaviours'...